cellxgene_schema_cli/cellxgene_schema/ontology.py

```diff
@@ -0,0 +1,280 @@
+import os
+import gzip
+import json
+import enum
+from typing import Union, List, Set
+from . import env
+
+
+class SupportedOrganisms(enum.Enum):
+    Homo_sapiens = 1
```

Comment:
```suggestion
    HOMO_SAPIENS = "HomoSapiens"
```

Reply:
Fixed in c973a8971d9f13dad5ab1606e5819f8ba41eb2cc